interface/forms/misc_billing_options/new.php

```diff
@@ -69,7 +67,7 @@
     </script>
   </head>
   <body class="body_top">
-    <form method='post' <?php echo "name='my_form' " . "action='$rootdir/forms/misc_billing_options/save.php?id=" . attr($formid) . "'>\n";?>>
```

Comment:
interesting, I wonder where it is coming from(wonder if browser is "fixing" it). If you add it here, are there then 2?